client/my-sites/plans/jetpack-plans/product-store/product-simple-card/index.tsx

```diff
@@ -0,0 +1,58 @@
+import { Button } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import { ItemPrice } from '../item-price';
+import { ProductSimpleCardProps } from '../types';
+import getProductIcon from '../utils/get-product-icon';
+
+import './style.scss';
+
+const ProductSimpleCard: React.FC< ProductSimpleCardProps > = ( {
+	item,
+	isOwned,
+	siteId,
+	onClickMore,
+	onClickPurchase,
+	checkoutURL,
+} ) => {
+	const translate = useTranslate();
+	const { shortName: name, description } = item;
+
+	return (
+		<div className="product-simple-card">
+			<div className="product-simple-card__icon">
+				<img alt="" src={ getProductIcon( { productSlug: item.productSlug } ) } />
+			</div>
+			<div className="product-simple-card__info">
+				<div className="product-simple-card__info-header">
+					<div className="product-simple-card__info-header-content">
+						<h3 className="product-simple-card__info-header-text">{ name }</h3>
+
+						<ItemPrice isOwned={ isOwned } item={ item } siteId={ siteId } />
+					</div>
+					<Button
+						className="product-simple-card__info-header-checkout"
+						onClick={ onClickPurchase }
+						href={ checkoutURL }
+						primary
+						compact
+					>
+						{ translate( 'Get' ) }
+					</Button>
+				</div>
+				<div className="product-simple-card__info-content">
+					{ description }
+					<Button
+						className="product-simple-card__info-more-link"
+						onClick={ onClickMore }
+						href="https://jetpack.com/pricing/"
```

Comment:
```suggestion
						href="#"
```

Reply:
this is more appropriate.. done changing it.. 👍